src/API/Management/Blacklists.php

```diff
@@ -15,7 +15,11 @@ public function getAll($aud)
     {
         $response = $this->httpClient->get('/blacklists/tokens?'.http_build_query(['aud' => $aud]));
 
-        return ResponseMediator::getContent($response);
+        if (200 === $response->getStatusCode()) {
```

Comment:
It feels a bit weird to me to be checking for a `statusCode`value here. Feels like the `ResponseMediator` should be doing more work.  @glena any thoughts on this?